apps/meteor/app/lib/server/startup/oAuthServicesUpdate.js

```diff
@@ -194,10 +194,10 @@ function customOAuthServicesInit() {
 					avatarField: process.env[`${serviceKey}_avatar_field`],
 				};
 
-				addOAuthService(name, values);
+				await addOAuthService(name, values);
 			}
 		}
-	});
+	}
 }
 
-customOAuthServicesInit();
+await customOAuthServicesInit();
```

Comment:
Would not be better to create a function as you did in the other ones? 🤔

Reply:
Ah, this is already a function 😬 but didn't move to the other places as this has some extra logic in there (getting settings, env, creating oauth classes etc) so left it for a future refactor: separate setting creation logic from oauth logic and break down the file